### PR TITLE
New package: i3lockr-0.1.0

### PR DESCRIPTION
**Pull request by sinetoami**

### CODE REVIEW

File: srcpkgs/i3lockr/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'i3lockr'
+pkgname=i3lockr
+version=0.1.0
+revision=1
+build_style=cargo
+makedepends="rust xcb-util-image-devel python3"
```
Comment (maxice8) [inline srcpkgs/i3lockr/template]: remove rust from makedepends

File: srcpkgs/i3lockr/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+short_desc="Distort a screenshot and run i3lock"
+maintainer="Sinésio Neto <sinetoami@gmail.com>"
+license="MIT and APACHE"
```
Comment (maxice8) [inline srcpkgs/i3lockr/template]: No, this is not how you declare multiple licenses. and APACHE doesn't follow spdx short-identifier

File: srcpkgs/i3lockr/template [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'i3lockr'
+pkgname=i3lockr
+version=0.1.0
+revision=1
+build_style=cargo
+makedepends="xcb-util-image-devel python3"
```
Comment (maxice8) [inline srcpkgs/i3lockr/template]: python3 needs to be in hostmakedepends=

Comment (sinetoami) [reply srcpkgs/i3lockr/template]: Is this the reason for the build fails?

**Comment by maxice8** [comment]
I guess yes, you should read the Travis build logs and fix the errors

**Comment by maxice8** [comment]
On Sun, Jan 27, 2019 at 09:28:45AM -0800, Sinésio Neto wrote:
> Sorry. So, Can you explain to me what the difference between `host` and `target` dependencies? I could not understand reading the manual. Thanks
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/7856#issuecomment-457937283

host dependencies are dependencies installed on /usr , they are of the same CPU architecture as the host system (the system that runs all the tools like gcc, cargo, python3).

host dependencies are meant for tooling that you need to run.


target dependencies are installed on /usr/$triplet/usr. triplet is a word that identifiers which CPU architecture the target system is. The target architecture can be of a different CPU architecture (in which case we are cross compiling).

the target dependencies are meant for libraries that you need to link your packages against.

File: srcpkgs/i3lockr/template [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+short_desc="Distort a screenshot and run i3lock"
+maintainer="Sinésio Neto <sinetoami@gmail.com>"
+license="MIT, Apache-2.0"
+homepage="https://github.com/owenthewizard/i3lockr"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=fc715687e21fd36f8927f45a5a6fbc04bf81e2e7bcbfa9866a511400551af314
+
+post_install() {
+	vlicense LICENSE-MIT.md
+	vlicense LICENSE-APACHE.md
```
Comment (maxice8) [inline srcpkgs/i3lockr/template]: don't need to vlicense APACHE, only MIT

Comment (sinetoami) [reply srcpkgs/i3lockr/template]: In the `license` variable, Do I have to take it off too?

**Comment by maxice8** [comment]
On Sun, Jan 27, 2019 at 12:30:33PM -0800, Sinésio Neto wrote:
> sinetoami commented on this pull request.
> 
> 
> 
> > +pkgname=i3lockr
> +version=0.1.0
> +revision=1
> +build_style=cargo
> +hostmakedepends="python3"
> +makedepends="xcb-util-image-devel"
> +short_desc="Distort a screenshot and run i3lock"
> +maintainer="Sinésio Neto <sinetoami@gmail.com>"
> +license="MIT, Apache-2.0"
> +homepage="https://github.com/owenthewizard/i3lockr"
> +distfiles="${homepage}/archive/v${version}.tar.gz"
> +checksum=fc715687e21fd36f8927f45a5a6fbc04bf81e2e7bcbfa9866a511400551af314
> +
> +post_install() {
> +	vlicense LICENSE-MIT.md
> +	vlicense LICENSE-APACHE.md
> 
> In the `license` variable, Do I have to take it off too?
> 
> -- 
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/7856#discussion_r251253877

No, you need to include the LICENSE file for certain licenses like MIT, ISC. You use the vlicense function for that. But not all licenses need their files present.

**Comment by sinetoami** [comment]
> No, you need to include the LICENSE file for certain licenses like MIT, ISC. You use the vlicense function for that. But not all licenses need their files present.

Done. :+1:

**Pull request closed without merging**